build-system/tasks/runtime-test.js

```diff
@@ -173,6 +176,7 @@ function printArgvMessages() {
         cyan(argv.grep) + '".',
     coverage: 'Running tests in code coverage mode.',
     headless: 'Running tests in a headless Chrome window.',
+    'chrome_canary': 'Running tests on Chrome Canary.',
```

Comment:
nit: For easy maintenance, move this up to the top, where the other local browser options are listed.

Reply:
Done.